sdk/containerregistry/azure-containers-containerregistry/src/test/java/com/azure/containers/containerregistry/ContainerRegistryClientIntegrationTests.java

```diff
@@ -176,19 +178,22 @@ public void convenienceProperties(HttpClient httpClient) {
     @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
     @MethodSource("getHttpClients")
     public void authenticationScopeTest(HttpClient httpClient) {
-        ContainerRegistryClient registryClient = getContainerRegistryBuilder(httpClient)
-            .authenticationScope(AZURE_GLOBAL_AUTHENTICATION_SCOPE)
-            .buildClient();
+        String authority = getAuthority(REGISTRY_ENDPOINT);
+        if (authority == AzureAuthorityHosts.AZURE_PUBLIC_CLOUD) {
+            ContainerRegistryClient registryClient = getContainerRegistryBuilder(httpClient)
+                .authenticationScope(AZURE_GLOBAL_AUTHENTICATION_SCOPE)
+                .buildClient();
 
-        List<String> repositories = registryClient.listRepositoryNames().stream().collect(Collectors.toList());
-        validateRepositories(repositories);
+            List<String> repositories = registryClient.listRepositoryNames().stream().collect(Collectors.toList());
+            validateRepositories(repositories);
 
-        if (getTestMode() != TestMode.PLAYBACK) {
-            // Now doing the same should fail with the separate registryClient;
-            ContainerRegistryClient throwableRegistryClient = getContainerRegistryBuilder(httpClient)
-                .authenticationScope(AZURE_GOV_AUTHENTICATION_SCOPE)
-                .buildClient();
-            assertThrows(ClientAuthenticationException.class, () -> throwableRegistryClient.listRepositoryNames().stream().collect(Collectors.toList()));
+            if (getTestMode() != TestMode.PLAYBACK) {
+                // Now doing the same should fail with the separate registryClient;
+                ContainerRegistryClient throwableRegistryClient = getContainerRegistryBuilder(httpClient)
+                    .authenticationScope(AZURE_GOV_AUTHENTICATION_SCOPE)
+                    .buildClient();
+                assertThrows(ClientAuthenticationException.class, () -> throwableRegistryClient.listRepositoryNames().stream().collect(Collectors.toList()));
+            }
```

Comment:
Should this be made into a separate test that has an assumption of `TestMode != PLAYBACK`